packages/eslint-config-airbnb/rules/react.js

```diff
@@ -9,7 +9,7 @@ module.exports = {
     // Prevent missing displayName in a React component definition
     'react/display-name': 0,
     // Enforce boolean attributes notation in JSX
-    'react/jsx-boolean-value': 2,
+    'react/jsx-boolean-value': [2, 'never'],
```

Comment:
Ah, the classic Double-J communication breakdown.

https://www.youtube.com/watch?v=f6WQLvRvtjs